scopes/component/lister/list.cmd.ts

```diff
@@ -16,16 +16,16 @@ type ListFlags = {
 
 export class ListCmd implements Command {
   name = 'list [remote-scope]';
-  description = 'list components on a workspace, local scope or a remote scope.';
+  description = 'list components on a workspace, local scope or a remote scope.'; // @david what's the difference between workspac and local scope? Can we just remove local scope in this description?
```

Comment:
@GiladShoham also commented about that
But when i run `bit list` which do i get as default? I'm guessing it's the workspace, so that's what should be added here.
There's a flag for showing the scope (which is advanced data, so it's on a need-to-use basis) so that shouldn't be in the main description.

Adding it makes this complicated - we should simplify as much as possible, and only include in the description features which are the main utility of the command (flags can be mentioned, but only when they're more mainstream use)

Reply:
I agree. You can remove the `local scope` from the description. It's not relevant for the majority of the cases.

(just a bit of history, in the past, this command wasn't relevant to the workspace at all.  It was only showing scope components)

Reply:
ok great, done